common/translations/markup.js

```diff
@@ -4,19 +4,28 @@ import React from 'react';
 const BOLD_REGEXP = /(\*\*)(.*?)\1/;
 const LINK_REGEXP = /\[([^\[]+)\]\(([^\)]+)\)/;
 
+function decodeHTMLEntities(text) {
```

Comment:
We just need to handle everything that is in translation files. We want to move to something else after release anyway.